tests/util/mod.rs

```diff
@@ -0,0 +1,5 @@
+// Not every test module -- which are compiled at seperate binaries, use every function, causing dead_code to be emitted.
+#![allow(dead_code)]
+
+pub mod dura;
```

Comment:
Hey question. Cargo is complaining about unused functions because snapshots_test doesn't use Dura. Is there a way to fix or suppress those warnings? Seems like it's not feasible to use all functions in every test.